verifier/util/util.go

```diff
@@ -0,0 +1,78 @@
+// Package util provides helper funtions to prepare materials for talking to attestation verifiers.
```

Comment:
I wonder if everything in util should just go in `internal` as well. I'm not sure this is valuable to export at the moment.